docs/modules/ROOT/pages/archetype.adoc

```diff
@@ -0,0 +1,142 @@
+= Creating a Project
+
+== Mapping to aiSSEMBLE(TM) Concepts
+[#img-you-are-here-archetype]
+.xref:solution-baseline-process.adoc[You Are Here]
+image::you-are-here-archetype.png[You Are Here,200,100,role="thumb right"]
+
+_Step 1: Instantiate a New aiSSEMBLE Project:_ The aiSSEMBLE archetype is an executable pattern that can quickly incept
+a project that is ready to leverage aiSSEMBLE for constructing components tailored to your needs. It is realized below
+using a Maven archetype feature.
+
+== Use a Maven Archetype to Create a New Project
+The first step in creating a new project is to leverage Maven's archetype functionality to incept a new Maven project 
+that will contain all of your aiSSEMBLE component implementations - data delivery and machine learning pipelines as well as
+path to production modules.
+
+Open a terminal to the location in which you want your project to live and execute the following command:
+[source]
+[subs=attributes+]
+
+ifeval::[{is-pre-release} == true]
+-- 
+mvn archetype:generate \
+	-DarchetypeGroupId=com.boozallen.aissemble \
+	-DarchetypeArtifactId=foundation-archetype \
+	-DarchetypeVersion={page-version}-SNAPSHOT
+--
+endif::[]
+
+ifeval::[{is-pre-release} != true]
+-- 
+mvn archetype:generate \
+	-DarchetypeGroupId=com.boozallen.aissemble \
+	-DarchetypeArtifactId=foundation-archetype \
+	-DarchetypeVersion={page-version}
+--
+endif::[]
+
+This command will trigger an interactive questionnaire giving you the opportunity to enter the following information
+(in alphabetical order) except if the option is not interactive (see table). For those options that are not interactive, they must be supplied to the ``mvn
+archetype:generate`` command by using the following format ``-D<propertyName>=<propertyValue>``. For those options that are interactive,
+there is also an option to supply the values in the same way.
+
```

Comment:
S: The wording here is really confusing. Maybe restructure to something like this:
This command will trigger an interactive questionnaire giving you the opportunity to enter the following information contained in the below table.
- This information can also be provided without interaction by including `-D<propertyName>=<propertyValue>` for each property you want to manually set when running the `mvn archetype:generate` command.
- Some of the information required (such as archetypeVersion) is unavailable in interactive mode. For these properties, you can use the above Maven command to include them.

Reply:
Looks good - ✅ INCORPORATED